maybe/syscall_filters.py

```diff
@@ -95,6 +96,17 @@ def format_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
+    path = abspath(path)
+    if (type & S_IFREG):
```

Comment:
Just like in `format_open`, we should only log if an actual action would be taken, that is if the path does not exist (as otherwise the call will simply fail).